.gitignore

```diff
@@ -34,6 +35,8 @@ release/
 .idea
 
 # Generated during build
-backend-assets/
+backend-assets/*
+core/http/backend-assets/
```

Comment:
I didn't had a closer look at this yet, but we have to be careful to not end up by skipping the built-in backend-assets. In the tests, IIRC we add that explictly so it skips the standard build process, so I'd like to test this manually first and checking it out locally before merging

Reply:
This was only to try and avoid an annoying lint popup on my side. If you think it's risky, I'm more than happy to drop it for now and test it later.

Reply:
Leaving this as-is in tonight's version, since I need to read up more on what this even is doing before I decide either way. Feel free to suggest a change here if you know what to do :D

Reply:
Ok, I'm still not an expert on embedded golang FS yet... but I think the right answer here is to remove the weird `core/http/backend-assets/` line I added to try and help `api_tests.go`, so I've done that.